packages/collector/Collector_Db.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+function collector_esc_sql_identifier($tableName) {
+	return str_replace('`', '``', esc_sql($tableName));
+}
+
+function collector_dump_db($zip)
+{
+	$tables   = collector_get_db_tables();
+	$sql_dump = '';
+
+	$sql_dump .= sprintf("-- %s\n", json_encode(['SECTION START' => 'SCHEMA']));
+
+	foreach($tables as $table)
+	{
+		$sql_dump .= sprintf("-- %s\n", json_encode(['ACTION' => 'DROP', 'TABLE' => $table]));
+		$sql_dump .= sprintf("DROP TABLE IF EXISTS `%s`;\n", collector_esc_sql_identifier($table));
+		$sql_dump .= sprintf("-- %s\n", json_encode(['ACTION' => 'CREATE', 'TABLE' => $table]));
+		$sql_dump .= preg_replace("/\s+/", " ", collector_dump_db_schema($table)) . "\n";
```

Comment:
Why replace the whitespace here? What if a default column value contains two spaces?